_layouts/home.html

```diff
@@ -12,7 +12,7 @@
 <body>
     <div id="particles-js"></div>
     <main>
-        <img class="home-logo" src="/assets/wordmark-web.png">
+        <img class="home-logo" src="/assets/wordmark-web.png" onclick="countClicks()">
```

Comment:
Thank you. Haven't tested, but the code looks okay. Approving, I'll wait for @HackerManiac to approve before merging.